app/src/main/java/com/github/wnder/tour/FirebaseTourDatabase.java

```diff
@@ -0,0 +1,235 @@
+package com.github.wnder.tour;
+
+import android.content.Context;
+import android.location.Location;
+
+import com.github.wnder.picture.FirebasePicturesDatabase;
+import com.google.firebase.firestore.CollectionReference;
+import com.google.firebase.firestore.FirebaseFirestore;
+import com.google.firebase.firestore.SetOptions;
+
+import java.util.Calendar;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+
+public class FirebaseTourDatabase implements TourDatabase{
+
+    private FirebasePicturesDatabase db;
+
+    private final CollectionReference tourCollection;
+
+    public FirebaseTourDatabase(Context context) {
+        tourCollection = FirebaseFirestore.getInstance().collection("tours");
+        db = new FirebasePicturesDatabase(context);
+    }
+
+    @Override
+    public CompletableFuture<List<String>> getTourPics(String tourUniqueId) {
+        CompletableFuture<List<String>> tour = new CompletableFuture<>();
+
+        tourCollection.document(tourUniqueId).get().
```

Comment:
Nit: move period to next line for consistency with the rest of the code

Reply:
done